test/compilable/imports/test15966base.d

```diff
@@ -0,0 +1,8 @@
+module pkg.test15966base;
+
+class Base
+{
+    public import imports.test15966a;
+    protected import imports.test15966b;
+    package(pkg) import imports.test15966c;
+}
```

Comment:
Could you add a test checking that private imports are visible within the same module ?